rcl_interfaces/msg/LoggerLevel.msg

```diff
@@ -0,0 +1,14 @@
+# Logger level
+uint8 LOG_LEVEL_UNKNOWN = 0
+uint8 LOG_LEVEL_DEBUG = 10
+uint8 LOG_LEVEL_INFO = 20
+uint8 LOG_LEVEL_WARN = 30
+uint8 LOG_LEVEL_ERROR = 40
+uint8 LOG_LEVEL_FATAL = 50
+
+# The logger name.
+string name
+
+# logger level: UNKNOWN, DEBUG, INFO, WARN, ERROR, FATAL
+#                  0       10    20    30     40     50
```

Comment:
```suggestion
```

These are now specified as constants, so the comment isn't needed

Reply:
Okay, I remove them.